packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -176,15 +176,16 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
           this.viewportData.rows[r - this.viewportData.offset];
         assertNotNull(intersection.startColumn);
         assertNotNull(intersection.endColumn);
-        for (
-          let c = intersection.startColumn;
-          c <= intersection.endColumn;
-          c += 1
-        ) {
-          assertNotNull(formatValue);
-          resultRow.push(
-            formatValue(viewportRow.data.get(c)?.value, this.columns[c])
-          );
+        if (formatValue != null) {
+          for (
+            let c = intersection.startColumn;
+            c <= intersection.endColumn;
+            c += 1
+          ) {
+            resultRow.push(
+              formatValue(viewportRow.data.get(c)?.value, this.columns[c])
```

Comment:
We still want to populate the `resultRow` array with unformatted values if `formatValue` is not defined.
With this change, the `resultRow` ends up empty.